src/lib/HighlightRenderer.ts

```diff
@@ -54,17 +54,25 @@ export class HighlightRenderer implements IRenderer {
       const newHighlightByNode = new Map<HTMLElement, HTMLElement>();
 
       for (const node of nodes) {
-        const rect = node.getBoundingClientRect();
-        const overlapped = isNodeOverlapped(node, rect);
-        const flat = rect.height === 0 || rect.width === 0;
-
-        if (!overlapped && !flat) {
-          const highlight = this.highlightByNode.get(node) ?? this.createHighlightForNode();
-
-          highlight.style.top = `${rect.top + window.pageYOffset}px`;
-          highlight.style.left = `${rect.left + window.pageXOffset}px`;
-          highlight.style.width = `${rect.width}px`;
-          highlight.style.height = `${rect.height}px`;
+        const nodeRect = node.getBoundingClientRect();
+        const isOverlapped = isNodeOverlapped(node, nodeRect);
+        const isFlat = nodeRect.height === 0 || nodeRect.width === 0;
```

Comment:
No, it is not related to the empty text, I discussed that problem with UX and they said that we don't need to fix that.
This check turns off highlighting for elements that are "flat" (width or/and height is 0). 
I tried adding several different elements to the SafeLife project and tested how the SDK works with them, and if an element is flat the highlight is displayed as a line and can never be hovered (because the element is basically invisible (has not height/width)). That is why I decided to add this check.

Reply:
I see, `isInvisible` or `hasZeroArea` would probably be better here but feel free to leave it